### PR TITLE
BIGTOP-3459. Deploying GPDB fails on ARM64 CentOS 7.

### PR DESCRIPTION
**Pull request by sekikn**

### CODE REVIEW

**Comment by sekikn** [comment]
Tested with CentOS 7, 8 and Fedora 31 on both aarch64 and x86_64.

File: bigtop-deploy/puppet/modules/gpdb/manifests/init.pp [hunk 1]
```diff
@@ -137,6 +140,16 @@
             ensure   => latest,
           }
           package { ["paramiko"]:
+            ensure   => latest,
+            provider => pip,
+            require  => Package["setuptools"],
+          }
+          package { ["setuptools"]:
```
Comment (elukey) [inline bigtop-deploy/puppet/modules/gpdb/manifests/init.pp:147]: Totally ignorant about the environment but are those packages named python-setuptools or simply setuptools?

**Comment by elukey** [comment]
LGTM 👍

**Comment by sekikn** [comment]
Thanks! Merged into master and branch-1.5.

**Pull request merged**